.gitignore

```diff
@@ -8,3 +8,7 @@
 out
 gen
 .DS_Store
+
+/node_modules/
+# Ignore this for now
+/package-lock.json
```

Comment:
For consistent behavior, I prefer locking dependencies with `package-lock.json` and install packages/tools with `npm install` both in CI and local development. 
What do you think? cc @yikeke @aylei

Reply:
IIRC, `npm install` may update `package-lock.json` @YiniXu9506 Could you please take a look?

Reply:
@aylei Yes! Looking on it.

Reply:
@g1eny0ung PALT, thanks!

Reply:
@cofyc Agree with u. Generally, it's necessary to keep `package-lock.json` or `yarn.lock` file if you don't want NPM packages updated unexpectedly.
For example, a node project may include a package which version is `^1.0.1`,  if we won't keep lock file, the next time running `npm i` or `yarn`, the actual version used may be `1.0.5`.

ref: https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json

Reply:
Yes, lock files should be checked in. @cofyc said though that users might get confused by seeing node_modules and package.json so I thought it might be better to just ignore both for now. But I am happy to check it in.

Reply:
checked in now